sdk/resourcemanager/azure-resourcemanager-resources/src/main/java/com/azure/resourcemanager/resources/fluentcore/exception/AggregatedManagementException.java

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.resourcemanager.resources.fluentcore.exception;
+
+import com.azure.core.management.exception.ManagementException;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+/**
+ * Exception thrown for an invalid response with custom error information.
+ */
+public final class AggregatedManagementException extends ManagementException {
+
+    /**
+     * Creates an instance of AggregatedManagementException.
+     *
+     * @param aggregatedException the aggregated exception.
+     * @param firstManagementException the first ManagementException in suppressed.
+     */
+    private AggregatedManagementException(RuntimeException aggregatedException,
+                                          ManagementException firstManagementException) {
```

Comment:
Is it better to add all ManagementException rather than the first one?

Reply:
All is added to suppressed list. There is only 1 "value" and "response" there, hence only point it to first ManagementException.

Reply:
Could user access suppressed?

Reply:
"suppressed" are in base Throwable https://docs.oracle.com/javase/7/docs/api/java/lang/Throwable.html#getSuppressed(), so that is common for all exceptions.